### PR TITLE
Convert expression to holonomic

### PR DESCRIPTION
**Pull request by shubhamtibra**

### CODE REVIEW

**Comment by shubhamtibra** [comment]
I have added a basic method using Look-Up table to convert Functions. Will add more stuff soon. Ping @jksuom @certik .

**Comment by shubhamtibra** [comment]
I have added few elementary functions and their holonomic representation in a dictionary and method to convert using this table, how it looks to you?

**Comment by shubhamtibra** [comment]
I have added `sin, cos, exp, log, erf` to the table. Let me know your views. Ping @jksuom @certik .

**Comment by certik** [comment]
@shubhamtibra this looks great to me. When you want this merged, remove the WIP and ping me

**Comment by shubhamtibra** [comment]
Conflicts are resolved now. Ping @certik @jksuom .

**Comment by jksuom** [comment]
I think this is ready. Merging.

**Comment by asmeurer** [comment]
This pull request has resulted in code quality tests failing [again](https://github.com/sympy/sympy/issues/11204). We really need to get this issue fixed, but in the meantime, @shubhamtibra, please configure your editor to clear trailing whitespace when you save files. What text editor do you use?

File: sympy/holonomic/holonomic.py [hunk 1]
```diff
@@ -1229,6 +1160,200 @@ def _find_conditions(simp, x, x0, order, evalf=False):
     return HolonomicFunction(sol, x).composition(z)
 
 
+x_1 = Dummy('x_1')
+_lookup_table = None
+from sympy.integrals.meijerint import _mytype
+
+
+def from_sympy(func, x=None, initcond=True):
```
Comment (asmeurer) [inline sympy/holonomic/holonomic.py]: These names will need to change, or else become methods of a class, if they are ever going to be added to a public namespace.

**Comment by certik** [comment]
Tests pass, so I don't understand how the trailing white space got in. Either way, @shubhamtibra, make sure you configure your editor to highlight the trailing white space as you type. For vim, put this into your `~/.vimrc`:
```
if has('gui_running')
   hi WhiteSpaceEOL guibg=#FF0000
else
   hi WhiteSpaceEOL ctermbg=Red
endif
match WhitespaceEOL /\s\+\%#\@<!$/
```

**Comment by shubhamtibra** [comment]
Sorry for the inconvenience caused, I use Sublime Text 3 and will take care of this thing from now on.

**Pull request merged**